checkST2110.js

```diff
@@ -101,7 +101,8 @@ a=ts-refclk:ptp=IEEE1588-2008:08-00-11-FF-FE-22-91-3C:127
 a=mediaclk:direct=0 
 a=mid:SECONDARY`;
 
-// ST 2110-10 Section 7.4 Test 1 - Where mediaclk:direct is used with PTP, offset value is zero
+// ST 2110-10:2017 Section 7.4
+// ST 2110-10:2022 Section 8.3 Test 1 - Where mediaclk:direct is used with PTP, offset value is zero
```

Comment:
I think since the name of the test function still follows the 2017 spec, better to move the "Test 1..." description to the first line?